docs/takeaway.rb

```diff
@@ -0,0 +1,60 @@
+class Takeaway
+
+attr_reader :table_order
+
+  def initialize
+    @table_order = Array.new
+  end
+
+  def show_menu
+    puts full_menu
+  end
+
+  def order(*items)
+    items.each do |list|
+      @table_order << list
+    end
+  end
+
+  def bill
+    calculate_bill
+  end
+
+  def place_order
+    "Thank you! Your order was placed and will be delivered before #{(Time.new + 3600)}"
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)